pkg/controller/migmigration/annotation.go

```diff
@@ -160,8 +181,9 @@ func (t *Task) getResticVolumes(client k8sclient.Client, pod corev1.Pod) ([]stri
 }
 
 // Add label to namespaces
-func (t *Task) labelNamespaces(client k8sclient.Client) error {
-	for _, ns := range t.sourceNamespaces() {
+func (t *Task) labelNamespaces(client k8sclient.Client, itemsUpdated int) (int, error) {
```

Comment:
Question : I am just curious, I don't think I completely understand the logic. Why pass `itemsUpdated` to each of these individual functions? Can't they just report the number of items they updated?

Reply:
Passing `itemsUpdated` makes sure that we get an update after every 50th object is processed. We can report the updated items within a function after a fixed number of objects are updated without passing the variable (this was originally the proposed solution), but this approach is implemented based on the feedback I received from the team.

Reply:
@pranavgaikwad the idea here is to make sure that we process 50 items total per reconcile.